modules/apps/staging/staging-bar-web/src/main/resources/META-INF/resources/view_layout_revision_details.jsp

```diff
@@ -149,144 +166,33 @@ else {
+		<portlet:param name="mvcPath" value="/view_layout_revision_status.jsp" />
+	</portlet:renderURL>
 
-				<c:if test="<%= !layoutRevision.isIncomplete() && !layout.isTypeContent() %>">
-					<li>
-						<a class="dropdown-item" href="javascript:void(0);" id="manageLayoutRevisions" onclick="<portlet:namespace />openPageVariationsDialog();">
-							<liferay-ui:message key="page-variations" />
-						</a>
-					</li>
-					<li>
-						<a class="dropdown-item" href="javascript:Liferay.fire('<portlet:namespace />viewHistory', {layoutRevisionId: '<%= layoutRevision.getLayoutRevisionId() %>', layoutSetBranchId: '<%= layoutRevision.getLayoutSetBranchId() %>'}); void(0);" id="viewHistoryLink">
-							<liferay-ui:message key="history" />
-						</a>
-					</li>
-				</c:if>
+	<portlet:renderURL var="markAsReadyForPublicationURL" windowState="<%= LiferayWindowState.EXCLUSIVE.toString() %>">
+		<portlet:param name="mvcPath" value="/view_layout_revision_details.jsp" />
+	</portlet:renderURL>
 
-				<c:if test="<%= !hasWorkflowTask && !layout.isTypeContent() %>">
-					<c:if test="<%= !layoutRevision.isMajor() && (layoutRevision.getParentLayoutRevisionId() != LayoutRevisionConstants.DEFAULT_PARENT_LAYOUT_REVISION_ID) %>">
-						<li>
-							<a class="dropdown-item" href="javascript:Liferay.fire('<portlet:namespace />undo', {layoutRevisionId: '<%= layoutRevision.getLayoutRevisionId() %>', layoutSetBranchId: '<%= layoutRevision.getLayoutSetBranchId() %>'}); void(0);" id="undoLink">
-								<liferay-ui:message key="undo" />
-							</a>
-						</li>
-					</c:if>
-
-					<c:if test="<%= layoutRevision.hasChildren() %>">
-
-						<%
-						List<LayoutRevision> childLayoutRevisions = layoutRevision.getChildren();
-
-						LayoutRevision firstChildLayoutRevision = childLayoutRevisions.get(0);
-						%>
-
-						<c:if test="<%= firstChildLayoutRevision.isInactive() %>">
-							<li>
-								<a class="dropdown-item" href="javascript:Liferay.fire('<portlet:namespace />redo', {layoutRevisionId: '<%= firstChildLayoutRevision.getLayoutRevisionId() %>', layoutSetBranchId: '<%= firstChildLayoutRevision.getLayoutSetBranchId() %>'}); void(0);" id="redoLink">
-									<liferay-ui:message key="redo" />
-								</a>
-							</li>
-						</c:if>
-					</c:if>
-				</c:if>
-			</ul>
+	<li class="control-menu-nav-item">
+		<div class="d-none d-sm-block">
+			<clay:dropdown-menu
+				additionalProps='<%=
+					HashMapBuilder.<String, Object>put(
+						"layoutRevisionStatusURL", layoutRevisionStatusURL
+					).put(
+						"markAsReadyForPublicationURL", markAsReadyForPublicationURL
+					).build()
+				%>'
+				borderless="<%= true %>"
+				displayType="unstyled"
+				dropdownItems="<%= stagingBarDisplayContext.getDropdownItems(layout, layoutRevision, hasWorkflowTask, layoutSetBranch) %>"
+				icon="ellipsis-v"
+				id='<%= liferayPortletResponse.getNamespace() + "stagingVersionOptions" %>'
```

Comment:
If we are not using this id in the frontend we can remove it. The component should be identified with some `aria-label` or `label`.

Reply:
cc @BarbaraCabrera